.circleci/config.yml

```diff
@@ -34,11 +34,10 @@ jobs:
       - checkout
       - run:
           command: |
-            pip install --upgrade awscli
-            $(aws ecr get-login --region us-east-1 --no-include-email)
             docker build -t lubot .
-            docker tag lubot:latest $DOCKER_REPOSITORY:latest
-            docker push $DOCKER_REPOSITORY:latest
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Login simplified. These two variables were created in the circle ci environment.